features/shop/apply_percentage_discount_promotion_on_order.feature

```diff
@@ -0,0 +1,27 @@
+@cart
+Feature: Apply percentage discount promotion on order
+    In order to pay proper amount while buying promoted goods
+    As a Customer
+    I want to have promotions applied to my cart
+
+    Background:
+        Given the store is operating on a single channel in "France"
+        And the store has a product "PHP T-Shirt" priced at "€100.00"
+        And there is a promotion "Holiday promotion"
+        And it gives "20%" percentage discount to every order
+
+    @todo
+    Scenario: Applying percentage discount to order
+        When I add product "PHP T-Shirt" to the cart
+        Then my cart total should be "€80.00"
+        And my cart promotions should be "-€20.00"
+
+    @todo
+    Scenario: percentage discount does not affect the shipping fee
```

Comment:
As an admin user, if you wanted to discount shipping as well would you just add a secondary `action` (or whatever it's going to be called) to also discount the same percentage on shipping?